submissions/xT0PERx/js-dom/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="visuality.css" />
+    <title>My interactive side-menu</title>
+  </head>
+  <body>
+    <div class="container">
+      <nav class="menu">
+        <p>О компании :</p>
```

Comment:
Use heading HTML tags. [Read more about it.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)